bahmnicore-omod/src/main/resources/liquibase.xml

```diff
@@ -1377,7 +1392,7 @@
             set @answer_concept_id = 0;
             call add_concept(@concept_id, @concept_name_short_id,
             @concept_name_full_id,'LAB_REPORT','LAB_REPORT', 'Text', 'URL', true);
-            select @labresults_concept_id := concept_id, min(concept_id) from concept_name where name = 'LABRESULTS_CONCEPT';
+            select @labresults_concept_id := concept_id, min(concept_id) from concept_name where name = 'LABRESULTS_CONCEPT' GROUP BY concept_id;
```

Comment:
@samuelmale I don't recall the issue here. Could you explain quickly the symptom?

Reply:
Same case here :

`ERROR 1140 (42000): In aggregated query without GROUP BY, expression #1 of SELECT list contains nonaggregated column 'bahmni-4.concept_name.concept_id'; this is incompatible with sql_mode=only_full_group_by`

Reply:
And here as well @samuelmale , checksum has changed due to an update in your query. You should assign a new ID.

Reply:
Done.